LayoutTests/http/tests/workers/service/shownotification-invalid-data.html

```diff
@@ -0,0 +1,58 @@
+<head>
+<script src="/resources/js-test-pre.js"></script>
+<script src="/resources/notifications-test-pre.js"></script>
+<script>
+
+function failTheTest(msg)
+{
+    testFailed(msg);
+    if (testRunner)
+        testRunner.notifyDone();
```

Comment:
You shouldn't be calling testRunner.notifyDone() explicitly in a test using js-test.js. You should be calling `finishJSTest()`.